modules/compliance-tests/src/smithy4s/compliancetests/internals/ServerHttpComplianceTestCase.scala

```diff
@@ -110,19 +112,32 @@ private[compliancetests] class ServerHttpComplianceTestCase[
             .use { server =>
               server.orNotFound
                 .run(makeRequest(baseUri, testCase))
-                .attemptNarrow[NotImplementedError] *>
-                ce.timeout(inputDeferred.get, 1.second).flatMap { foundInput =>
-                  inputFromDocument
-                    .decode(testCase.params.getOrElse(Document.obj()))
-                    .liftTo[F]
-                    .map { decodedInput =>
-                      assert.eql(foundInput, decodedInput)
+                .attemptNarrow[NotImplementedError]
```

Comment:
While you're at it : it'd be good to change this to an error we control, something like 

```
case class IntendedShortCircuit() extends scala.util.control.NoStackTrace 
```

 or something. I opened your PR again this morning and was like "why are we doing this again ?" before I remembered.

Reply:
good idea